### PR TITLE
Add flower to the list 

### PR DESCRIPTION
**Pull request by Christykumar**
- [x] Add flower to the list feature added
- [x] Validations are tested

### CODE REVIEW

File: src/main/java/in/bloomapp/servlet/AddFlowerServlet.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import in.bloomapp.service.FlowerManager;
+
+/**
+ * Servlet implementation class AddFlowerServlet
+ */
+@WebServlet("/AddFlowerServlet")
+public class AddFlowerServlet extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+ 
+	@Override
```
Comment (selva08sankari) [inline src/main/java/in/bloomapp/servlet/AddFlowerServlet.java:21]: there is no comment lines in servlet

File: src/main/java/in/bloomapp/util/IsValid.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+		//Default constructor
+	}
+	/**
+	 * Checks weather the string is a valid string or not
+	 * @param checkString
+	 * @return
+	 */
+	public static boolean isValidString(String checkString) {
+	boolean validity=false;
+	if (!checkString.trim().equals("") || Pattern.matches("[a-zA-Z0-9]", checkString)) {
```
Comment (selva08sankari) [inline src/main/java/in/bloomapp/util/IsValid.java]: is this pattern allow space like( lotus flower)

File: src/main/webapp/addFlower.jsp [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+<body>
+<jsp:include page="header.jsp"></jsp:include>
+	<main class="container-fluid">
+		<h3>Add Flowers</h3>
+		
+		<form action="AddFlowerServlet" method="get">
+		
+		<label for="Category">Flower Category</label>
+		<input type="text" name="Category" placeholder="Enter Flower Category" required autofocus />
+		<br/>
```
Comment (selva08sankari) [inline src/main/webapp/addFlower.jsp:18]: where is the html validation

File: src/main/java/in/bloomapp/util/IsValid.java (hunk 2 above)
Comment (Christykumar) [reply src/main/java/in/bloomapp/util/IsValid.java]: Bug corrected

**Pull request merged**